process faults {7: ({4}, 327057016)}
network faults {}
{5: ("33BB84E4C5216A20EDBF94449DC1EAF66FECF79E6CFF664C1AC589038569AD62", {5, 6, 1, 0, 3, 2, 4}), 6: ("63969C90358E6FAE67C3D0784EA3F8648B8F64C55A2E2D9827111CE40484550A", {5, 4, 1, 0, 6, 3, 2}), 7: ("A88D8FCEE6A7326169E3ED104E81BBF423C9BD1C0AE3520E6A2785756B036291", {4, 1, 6, 3, 2, 0, 5}), 8: ("6AF924AF3760D1EE67AEFC90F44CE194F023A46CE2743708C92B220FE87EDA32", {6, 4, 5, 1, 3, 0, 2}), 10: ("0BD92D8C4E93067DFA3448358858B42960614195BAD6C6677753AD999285FBDE", {1, 2, 4, 6, 0, 5, 3}), 9: ("6056EDAEA899982B18D2B26E369CCEE6461905DDAA993E3CEA9AF76A3F5268BC", {0, 6, 4, 2, 1, 5, 3})}
7
reason: all committed
done!
